tests/infer/mcmc/test_hmc.py

```diff
@@ -25,20 +24,20 @@ def __init__(self, dim, chain_len, num_obs):
         self.dim = dim
         self.chain_len = chain_len
         self.num_obs = num_obs
-        self.mu_0 = Variable(torch.zeros_like(torch.Tensor(self.dim)), requires_grad=True)
-        self.lambda_prec = Variable(torch.ones_like(torch.Tensor(self.dim)))
+        self.mu_0 = torch.zeros_like(torch.tensor(self.dim))
```

Comment:
Why not simply `torch.zeros(self.dim)`?

Reply:
My bad, all of this can be simplified now, since we `torch.zeros/ones` support dytes.